server/build/webpack.js

```diff
@@ -104,22 +113,6 @@ export default async function createCompiler (dir, { hotReload = false, dev = fa
       name: 'dist/[path][name].[ext]'
     }
   }, {
-    loader: 'babel',
-    include: nextPagesDir,
-    query: {
-      sourceMaps: dev ? 'both' : false,
-      plugins: [
-        [
-          require.resolve('babel-plugin-module-resolver'),
-          {
-            alias: {
-              'ansi-html': require.resolve('ansi-html')
```

Comment:
We need handle `ansi-html` here, since we want to replace alias only on `nextPagesDir`